### PR TITLE
Completed all tasks

### PR DESCRIPTION
**Pull request by tautvilis**
Added fitler, sorting, order history, listing history etc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +22,7 @@
   "scripts": {
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
-    "start": "react-scripts start",
+    "start": "react-scripts start --host 0.0.0.0",
```
Comment (ElektrosStulpas) [inline package.json:25]: what does this do? a quick google search doesn't give an answer

File: src/Components/OrderHistory.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    
+    useEffect(() => {
+        if(!isPageLoaded){
+            setIsLoaded(true);
+        }
+    }, []);
+
+    useEffect(() => {
+
+    }, [listingHistory])
```
Comment (ElektrosStulpas) [inline src/Components/OrderHistory.js:90]: what's the point of this, just to reload page on listingHistory change?

**Review by ElektrosStulpas — COMMENTED** [review]
tbh didn't look very thoroughly as I'm more interested if this breaks what's already there. But generally looks good, few minor questions

**Review by accountas — APPROVED** [review]

**Comment by accountas** [comment]
LGTM

File: src/Components/OrderHistory.js (hunk 2 above)
Comment (tautvilis) [reply src/Components/OrderHistory.js:90]: Yes

File: package.json (hunk 1 above)
Comment (tautvilis) [reply package.json:25]: Bind the server to all ip's so it could be reached from WAN

**Pull request merged**